rise-storage.html

```diff
@@ -1051,8 +1051,13 @@
         this._hasAttemptedGetRequest = true;
       }
       else {
-        this._startTimer();
-        this.fire("rise-cache-error", resp);
+        // Get metadata for single file on a 502 response from Rise Cache.
+        if ((e.detail.request.xhr.response.status === 502) && this._isFile) {
+          this.getMetadata();
```

Comment:
@stulees It does. This call will eventually end up here - https://github.com/Rise-Vision/rise-storage/blob/763e8a757a81d9fee7ac789f3c6bfbe4be2952a0/rise-storage.html#L499